app/src/main/res/layout/details_screen.xml

```diff
@@ -4,39 +4,185 @@
     android:layout_height="match_parent"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    android:layout_margin="16dp">
+    android:background="@color/primary"    >
 
-    <ImageView
-        android:id="@+id/character_image"
+
+    <ImageButton
+        android:id="@+id/back"
+        android:layout_width="40dp"
+        android:layout_height="40dp"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toTopOf="@+id/character_image"
+        app:layout_constraintStart_toStartOf="parent"
+        android:backgroundTint="#00000000"
+        android:scaleType="centerCrop"
+        android:src="@drawable/backimg"
+        />
+
+
+
+
+   <!--
+    <Button
+        android:id="@+id/back_button"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:layout_margin="0dp"
+        android:backgroundTint="#00000000"
+        android:text="@string/back"
+        android:textColor="@color/black"
+        android:textSize="20dp"
+        android:textStyle="bold"
+        app:layout_constraintBottom_toTopOf="@+id/character_image"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+    -->
+
+    <ImageView
+        android:id="@+id/character_image"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        android:padding="16dp"
-        />
+        app:layout_constraintTop_toBottomOf="@+id/back"
+        app:layout_constraintWidth_percent="0.8"
+        app:layout_constraintDimensionRatio="1:1"
+        tools:src="@drawable/error" />
+
 
     <TextView
         android:id="@+id/character_name"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:padding="16dp"
-        tools:text="NombreTest"
+        android:paddingTop="4dp"
+        tools:text="Rick Sanchez"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/character_image"
+        android:textStyle="bold"
+        android:textSize="20dp"
         />
 
 
     <TextView
         android:id="@+id/character_status"
+        android:textColor="@android:color/darker_gray"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        tools:text="StatusTest"
+        tools:text="Alive"
         app:layout_constraintTop_toBottomOf="@id/character_name"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintEnd_toEndOf="parent"/>
 
 
+    <androidx.cardview.widget.CardView
+        android:id="@+id/cardlocation"
+        android:layout_width="match_parent"
```

Comment:
Why do you have 0dp in the image and match_parent here?

Reply:
Fixed, now it takes up 80% of the screen and is no longer match parent.